brat/Brat/Load.hs

```diff
@@ -153,7 +154,7 @@ loadStmtsWithEnv ns (venv, oldDecls, oldEndData) (fname, pre, stmts, cts) = addS
   let vdecls = map fst entries
   -- Now generate environment mapping usernames to nodes in the graph
   venv <- pure $ venv <> M.fromList [(name, overs) | ((name, _), (_, overs)) <- entries]
-  ((), (holes, newEndData, graph, _)) <- run venv kcStore ns $ withAliases aliases $ do
+  ((), (holes, newEndData, graph)) <- run venv kcStore newRoot $ withAliases aliases $ do
```

Comment:
It'd be more consistent with `"globals"` to do the split outside `run` here, or to use `-!` above (as the old code did), but I'm not too bothered if you want to be different!

Reply:
I don't think we need to change the split here, since we don't do anything with the new namespace